### PR TITLE
Bump version to v0.4.0

### PR DESCRIPTION
**Pull request by 2015aroras**

### CODE REVIEW

**Review by epwalsh — COMMENTED** [review]
Wait why wasn't this handled by the release script?

**Comment by epwalsh** [comment]
Ah, permissions issue I guess? I'll fix your permissions for future releases.

**Review by epwalsh — APPROVED** [review]

File: olmo/version.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 _MAJOR = "0"
-_MINOR = "3"
+_MINOR = "4"
 # On main and in a nightly release the patch should be one ahead of the last
 # released build.
 _PATCH = "1"
```
Comment (epwalsh) [inline olmo/version.py:5]:
```suggestion
_PATCH = "0"
```

Comment (2015aroras) [reply olmo/version.py]: I thought we kept the repo at version + 1 (so v0.4.1 when we are at 0.4.0)

**Comment by 2015aroras** [comment]
I ran the release script but I couldn't do it from main (maybe permissions)? It's not clear whether to do it from main or a branch

File: olmo/version.py (hunk 1 above)
Comment (epwalsh) [reply olmo/version.py]: No we typically leave it as is until the next release

**Comment by epwalsh** [comment]
> I ran the release script but I couldn't do it from main (maybe permissions)? It's not clear whether to do it from main or a branch

You should do it from main. I fixed your permissions so next time it should work for you.

**Pull request merged**